Library/Homebrew/test/test_options.rb

```diff
@@ -56,7 +56,7 @@ def test_equality
 
 class OptionsTests < Homebrew::TestCase
   def setup
-    @options = Options.new
+    @options = Options.create
```

Comment:
That syntax seems better to me. Another option is to have `Option.from_hash` and `Options.from_array` methods.



> As it's worthy to mention that Options.create has been there long before this PR.

Sure and it may be worth removing. My main complaints are that you're have to alter so many tests to use it and stating that it shouldn't be used without using anything like `private` to ensure it is not.